temoa/utilities/db_migration_v3_to_v3_1.py

```
@@ -307,14 +287,14 @@ def column_check(old_name: str, new_name: str) -> bool:
         for r, t in data[['region','tech']].drop_duplicates().values:
             periods[r, t] = [
                 p for p in time_optimize
-                if any((
+                if any(
                     v <= p < v+lifetime_process[r, t, v]
                     for v in [
                         t[0] for t in con_old.execute(
                             f'SELECT vintage FROM Efficiency WHERE region == "{r}" AND tech == "{t}"'
```

Ruff (0.14.0)

Possible SQL injection vector through string-based query construction

(S608)